src/popup/modules/InitQrCode.js

```diff
@@ -5,12 +5,12 @@ import * as MessageHandler from "/common/modules/MessageHandler.js";
 import * as QrCreator from "./QrCreator.js";
 import * as BrowserCommunication from "./BrowserCommunication.js";
 import * as UserInterface from "./UserInterface.js";
+import { retryPromise } from "../../common/modules/HelperFunctions.js";
```

Comment:
for common modules, you can use an absolute path (because that `..` does not bring any information value), just have a look how I did it in other modules.

Reply:
Hmm, is this part of the modules spec?

Reply:
Well... actually you can use path (without variables or so) in ES6 modules. That `/` is the root dir in WebExtensions, is actually a feature of WebExtensions (because you obviously have no website root dir).